doc/Development_Documentation/23_Installation_and_Upgrade/09_Upgrade_Notes/README.md

```diff
@@ -1,6 +1,7 @@
 # Upgrade Notes
 
 ## 10.6.0
+- [Storage config] Deprecated setting write targets and storage directory in the .env file. Instead, use the symfony config.
```

Comment:
Here we should link to the updated documentation and explain how to upgrade when still using env variables.